File: Que_05.py
# Check if a Number is Prime
# Input: 7 → Output: Prime

def is_prime(n):
    if n <= 1:
        return "Not Prime"
    
    for i in range(2, int(n ** 0.5) + 1):
        if n % i == 0:
            return "Not Prime"
        
    return "Prime"

print(is_prime(7))
    

File: Que_05.js
// Check if a Number is Prime
// Input: 7 → Output: Prime

function checkPrime(num){
    if (num <= 1) return "Not Prime";

    for (let i = 2; i < num; i++) {
        if (num % i === 0){
            return "Not Prime"
        };
        
    }

    return "Prime"
}

console.log(checkPrime(7));
